test/unit/conftest.py

```diff
@@ -0,0 +1,25 @@
+from collections.abc import Generator
+from unittest import mock
+
+import mocks
```

Comment:
I think this would be better as `from . import mocks` - it would get sorted properly (below `pytest`) and it's clearer that it's something that's in this project and not some third-party "mocks" package.

(Same comment for everywhere that has the import).

Reply:
Good call, done.